### PR TITLE
metrics, store/tikv: refine tikv client metrics

### PR DESCRIPTION
**Pull request by coocood**
Add region cache counter.

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: store/tikv/region_cache.go [hunk 1]
```diff
@@ -367,7 +369,7 @@ func (c *RegionCache) loadRegionByID(bo *Backoffer, regionID uint64) (*Region, e
 				return nil, errors.Trace(err)
 			}
 		}
-
+		metrics.TiKVRegionCacheCounter.WithLabelValues("get_region_by_id").Inc()
```
Comment (zimulala) [inline store/tikv/region_cache.go]: Do we need to add the result ?

**Comment by shenli** [comment]
@coocood Please address the comment.

**Comment by coocood** [comment]
@zimulala PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**